packages-ts/gauntlet-terra-contracts/codeIds/testnet-bombay.json

```diff
@@ -1,10 +1,10 @@
 {
-  "flags": 50280,
-  "ocr2": 50283,
-  "access_controller": 50284,
-  "deviation_flagging_validator": 50281,
+  "flags": 51109,
```

Comment:
Should also be done for `./testnet-bombay-internal.json` and all other per-network definitions in this folder.

Reply:
RIght forgot about that - I think `./testnet-bombay-internal.json` is the only since `mainnet` and `local` can't use the same codeIds